Assets/MixedRealityToolkit/_Core/Definitions/InteractionDefinitionMapping.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Definitions
+{
+    public struct InteractionDefinitionMapping
+    {
+        public InputType InputType { get; set; }
+        public AxisType AxisType { get; set; }
+        public InputAction InputAction { get; set; }
```

Comment:
I still feel like `InputType ==InputAction` because of their enum types.  Maybe we can change `InputAction` to a struct with some data that users can set per interaction, like the action id, a custom name, etc...

Reply:
As discussed above:   I think we resolved this in talk last night:

* InputType is the Physical Input Type (maybe we should rename it)
* Input Action is the Logical Action that is (should) be sent to the InputSystem

The mapping directs a specific input to output the action the dev has configured. Multiple inputs can point to the same action.